### PR TITLE
fix: remaining nfts

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Related #9390

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

### CODE REVIEW

**Comment by preschian** [comment]
@prury could you check, nft id 8 & 9 works on me. after that not able to mint https://github.com/kodadot/private-workers/issues/64 🤔

**Comment by prury** [comment]
image
forever loading on listing button

@preschian added some stuff to the issue, what should work on this PR?

**Comment by preschian** [comment]
> this is always 5/5, even if i already minted 2
> previews inside and outside are the same

@prury this one should be fixed

**Comment by prury** [comment]
i've minted 3, maximum is 5, there are 2 remaining, seems ok, let me make more tests

image

**Comment by preschian** [comment]
```
this message keeps appearing:

this is always 5/5, even if i already minted 2


no success/listing modal

previews inside and outside are the same
```

@prury @Jarsen136 updated, this issue is probably fixed. you can continue Jarsen from this branch. feel free to make a new PR, I can close this one if you create a new one

**Comment by prury** [comment]
- sometimes this happens: 

image

- tried to mint more than 5 and received error message: ✔️ 



- minted everything ✔️ 

image

- i had 5 complexity on CmWHiv7h4m9tEzKD94DH4mqwGTvsdYQe2nouWPF7ipmHpqA, minted them all and transferred one to DViifyZQ6cXAGE6KzPpEnNs3Gy1GC2nwujjicxTBfaXMs9C(it had 0) and the page says I'm eligible to mint: 

image

**Comment by preschian** [comment]
> * i had 5 complexity on CmWHiv7h4m9tEzKD94DH4mqwGTvsdYQe2nouWPF7ipmHpqA, minted them all and transferred one to DViifyZQ6cXAGE6KzPpEnNs3Gy1GC2nwujjicxTBfaXMs9C(it had 0) and the page says I'm eligible to mint:

oh, this is an interesting one

File: components/collection/drop/HolderOfGenerative.vue [hunk 1]
```diff
@@ -350,21 +397,25 @@ const submitMint = async (sn: string) => {
...
-  const nftIds = nftEntities.map((nft) => nft.sn)
+  const nftIds =
+    collectionData.value?.collectionEntity.nfts.map((nft) => nft.sn) || []
   const claimed = await Promise.all(
     nftIds.map((sn) => isNftClaimed(sn, holderOfCollectionId.value as string)),
   )
-  availableNfts.amount = claimed.filter((x) => !x).length
+  availableNfts.amount =
+    maxMintLimitForCurrentUser.value - claimed.filter((x) => !x).length
   availableNfts.isLoading = false
```
Comment (Jarsen136) [inline components/collection/drop/HolderOfGenerative.vue:419]: It is incorrect. The root issue is that `isNftClaimed` method does not work. https://github.com/kodadot/nft-gallery/issues/9404

**Comment by vikiival** [comment]
cc @prury

**Review by Jarsen136 — APPROVED** [review]

**Comment by Jarsen136** [comment]
> * sometimes this happens:
> 
> image
> 
> * tried to mint more than 5 and received error message: ✔️
> 
> 
> * minted everything ✔️
> 
> image
> 
> * i had 5 complexity on CmWHiv7h4m9tEzKD94DH4mqwGTvsdYQe2nouWPF7ipmHpqA, minted them all and transferred one to DViifyZQ6cXAGE6KzPpEnNs3Gy1GC2nwujjicxTBfaXMs9C(it had 0) and the page says I'm eligible to mint:
> 
> image

All of them should be fixed

**Comment by yangwao** [comment]
@prury this one is for release 😄

**Comment by prury** [comment]
> @prury it's so easy to do it
> Screen.Recording.2024-02-15.at.13.16.03.mov

lol waittt

**Comment by prury** [comment]
@yangwao let's go, found some other problems tho

**Comment by yangwao** [comment]
> found some other problems tho

severe?

**Comment by yangwao** [comment]
I think it works on 80% cases I'm good lol

**Comment by yangwao** [comment]
if we can launch it's good, no one needs to land back.

**Comment by prury** [comment]
> if we can launch it's good, no one needs to land back.
> 
>  [ 68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f6e3675546f4764306a3376646e4b437136332f67697068792e676966 ](https://camo.githubusercontent.com/b3191532d6ea5f5d5802800bceac3cdb6d5af49eacefbf0dd998e94f0b74385d/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f6e3675546f4764306a3376646e4b437136332f67697068792e676966) [ ](https://camo.githubusercontent.com/b3191532d6ea5f5d5802800bceac3cdb6d5af49eacefbf0dd998e94f0b74385d/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f6e3675546f4764306a3376646e4b437136332f67697068792e676966)

hehe, minor i guess: https://github.com/kodadot/nft-gallery/issues/9390#issuecomment-1945989005
but not from this PR

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 7.91 USD/DOT ~ 6.321 $DOT
🧗 [1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p](https://polkadot.subscan.io/account/1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)
🔗 [0x0aad85ac100021e5c928cf9547e73186e1d8c8a4f95b87e14c42778c6db4645d](https://polkadot.subscan.io/extrinsic/0x0aad85ac100021e5c928cf9547e73186e1d8c8a4f95b87e14c42778c6db4645d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**